src/mscorlib/src/System/Text/Normalization.cs

```diff
@@ -323,5 +324,27 @@ internal static String Normalize(String strInput, NormalizationForm normForm)
         [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
         unsafe private static extern void nativeNormalizationInitNormalization(
             NormalizationForm normForm, byte* pTableData);
+#else
+        unsafe private static int nativeNormalizationNormalizeString(
```

Comment:
what exactly the issue that @Jose Perez Rodriguez was looking at?

---
In reply to: [75737900](https://github.com/dotnet/coreclr/pull/6694#discussion_r75737900) [](ancestors = 75737900)